<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QApple</class>
 <widget class="QMainWindow" name="QApple">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QApple</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>S&amp;ystem</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionPause"/>
    <addaction name="actionReboot"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>&amp;Video</string>
    </property>
    <addaction name="actionBenchmark"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Wi&amp;ndow</string>
    </property>
    <addaction name="actionMemory"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Too&amp;ls</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuSystem"/>
   <addaction name="menuVideo"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action4_3"/>
   <addaction name="actionX1"/>
   <addaction name="actionX2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
  </action>
  <action name="action4_3">
   <property name="text">
    <string>4:3</string>
   </property>
  </action>
  <action name="actionX1">
   <property name="text">
    <string>x1</string>
   </property>
  </action>
  <action name="actionX2">
   <property name="text">
    <string>x2</string>
   </property>
  </action>
  <action name="actionReboot">
   <property name="text">
    <string>&amp;Reboot</string>
   </property>
  </action>
  <action name="actionBenchmark">
   <property name="text">
    <string>&amp;Benchmark</string>
   </property>
  </action>
  <action name="actionMemory">
   <property name="text">
    <string>&amp;Memory</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>&amp;Options...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
